### PR TITLE
[CodeStyle][CINN] fix cpplint codestyle `[build/include_order]` `[build/header_guard]` `[build/include_subdir]` `[build/include]`

### PR DESCRIPTION
**Pull request by enkilee**
### PR types

Others
### PR changes

Others
### Description

* https://github.com/PaddlePaddle/Paddle/issues/54953
fix CINN cpplint codestyle  `[build/include_order]` `[build/header_guard]` `[build/include_subdir]` `[build/include]`

### CODE REVIEW

File: paddle/cinn/frontend/computation.h [hunk 1]
```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#ifndef PADDLE_CINN_FRONTEND_COMPUTATION_H_
+#define PADDLE_CINN_FRONTEND_COMPUTATION_H_
```
Comment (SigureMo) [inline paddle/cinn/frontend/computation.h]: 用 `#pragma once` 呢？

**Comment by SigureMo** [comment]
流水线报错也需要看下

File: paddle/cinn/frontend/computation.h (hunk 1 above)
Comment (enkilee) [reply paddle/cinn/frontend/computation.h]: 我看了下paddle端的写法，有用#pragma once的，也有#ifndef的，不知道有没有规定用哪种。工具给出的建议是用#ifndef

Comment (SigureMo) [reply paddle/cinn/frontend/computation.h]: 看周围文件，不要一味地参考 Linter 的建议

Comment (enkilee) [reply paddle/cinn/frontend/computation.h]: 收到

**Review by SigureMo — APPROVED** [review]

**Pull request merged**